### PR TITLE
fix: blurry icons in messagelist

### LINKED ISSUE #1646: "Lock" and "√" icons appear blurry in message list
- Device:
- iOS version:
- Delta Chat version:
- Expected behavior:
- Actual behavior:
- Steps to reproduce the problem:
- Screenshots:
- Logs:

### PR DESCRIPTION
**Pull request by Simon-Laux**
fixes #1646

### CODE REVIEW

File: deltachat-ios/Assets.xcassets/padlock.imageset/Contents.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+      "filename" : "padlock.svg",
+      "idiom" : "universal",
+      "scale" : "3x"
+    }
+  ],
+  "info" : {
+    "author" : "xcode",
+    "version" : 1
+  }
+}
```
Comment (r10s) [inline deltachat-ios/Assets.xcassets/padlock.imageset/Contents.json:23]: imagesets do not support svg on older system we're still supporting. so we need to use bitmaps with single, double and triple size (as we're doing for all the other images)

in fact, the old `ic_lock` already is in this format, however, probably not in the optimal resolution

**Review by r10s — CHANGES_REQUESTED** [review]

File: deltachat-ios/Assets.xcassets/padlock.imageset/Contents.json (hunk 1 above)
Comment (Simon-Laux) [reply deltachat-ios/Assets.xcassets/padlock.imageset/Contents.json:23]: >  the old ic_lock already is in this format

yes, I noticed but it was always selecting the worst resolution when displaying. Do you know what is the minimum target for svg images icons?

Comment (r10s) [reply deltachat-ios/Assets.xcassets/padlock.imageset/Contents.json:23]: ios13 afaik

**Comment by r10s** [comment]
closing as it does not fix the issue, in general, svg is not the solution, even stock images appear blurry.

the issue probably is the NSTextAttachment, cmp https://stackoverflow.com/questions/27281945/blurry-image-nstextattachment-on-nsattributedstring

**Comment by Simon-Laux** [comment]
> closing as it does not fix the issue, in general, svg is not the solution, even stock images appear blurry.

I did not change the other icons, those worked without changing to svg.
Maybe we can just pick those changes of the pr and deal with the padlock later?

**Pull request closed without merging**